app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -235,8 +235,7 @@ public void onChanged(@Nullable String url) {
             @Override
             public void onValueChanged(@NonNull Boolean loading) {
                 if (loading) {
-                    backgroundTransitionGroup.resetTransition();
-
+                     backgroundTransitionGroup.resetTransition();
```

Comment:
Please don't make changes to unrelated code – it makes it harder to track the changes in code over time with `git blame`!